adafruit_pcf8591/pcf8591.py

```diff
@@ -85,10 +97,10 @@ def _half_read(self, channel):
         with self.i2c_device as i2c:
             i2c.write_then_readinto(self._buffer, self._buffer)
 
-    def read(self, channel):
+    def read(self, channel: Literal[0, 1, 2, 3]) -> None:
         """Read an analog value from one of the four ADC inputs
 
-        param: :channel The single-ended ADC channel to read from, 0 thru 3
+        param: int channel The single-ended ADC channel to read from, 0 thru 3
```

Comment:
Thanks!  The parameter should also be fixed to be `:param int channel: The single-ended ....` now so it renders properly by Sphinx in ReadTheDocs.

Reply:
yeah, docstrings are one of those things that I know of (more and more) but I haven't read the documentations regs yet to see how the build automation wants it.  maybe if I knew more I'd add more comments as I go, so needs to be soon on my list of things to do....